static/js/stream_create.js

```diff
@@ -359,6 +359,7 @@ $(function () {
                                       trigger: "manual"});
         announce_stream_docs.popover('show');
         announce_stream_docs.data('popover').tip().css('z-index', 2000);
+        announce_stream_docs.data('popover').tip().find('.popover-content').css('margin', '9px 14px');
```

Comment:
I think it might be better done by adding a class and some CSS to the actual popover template (possibly also removing the line above this to instead go there): `static/templates/announce_stream_docs.handlebars`

Reply:
Are you suggesting that I do something like `announce_stream_docs.data('popover').addClass('popover')` and have a `.popover` query changing the style of that popover?

Reply:
Nope, I'm suggesting editing the `static/templates/announce_stream_docs.handlebars` to add a CSS class to the popover, and then just do normal CSS styles in static/styles/ to style it. 

Basically, this model of hand-adding CSS through javascript is hacky and totally unnecessary, given that we just generated the HTML from a nice template 2 lines earlier (and this is the only place we use that HTML template).

Reply:
`static/templates/announce_stream_docs.handlebars` only contains the contents of the popover and can't modify the popover div surrounding it?

Reply:
Hmm, I see.  OK, I guess my idea doesn't work.